saleor/graphql/account/resolvers.py

```diff
@@ -50,18 +50,22 @@ def resolve_staff_users(info, query, **_kwargs):
     return qs.distinct()
 
 
-def resolve_user(info, id):
+def resolve_user(info, id=None, email=None):
     requester = get_user_or_app_from_context(info.context)
     if requester:
-        _model, user_pk = graphene.Node.from_global_id(id)
+        filterkwargs = {}
```

Comment:
In Python it should be `filter_kwargs`

Reply:
I actually named it like that inspired by how django calls it "initkwargs" and not "init_kwargs", but I've done what you said in the commit anyway